programs/pluto/whack_connectionstatus.c

```diff
@@ -826,7 +827,12 @@ static void show_connection_status(struct show *s, const struct connection *c)
 			jam_string(buf, "@");
 		}
 		if (oriented(c)) {
-			jam_string(buf, c->iface->real_device_name);
+			if (kernel_xfrmi_req_phy()) {
+				char *none = "NONE";
+				jam_string(buf, none);
+			} else {
+				jam_string(buf, c->iface->real_device_name);
+			}
```

Comment:
I'm fairly sure that this will show:
```
interface: NONE
```
when there's no IPsec Interface and a recent kernel.

Reply:
right. I pushed a fix.